docs/UserGuide.md

```diff
@@ -62,7 +62,15 @@ Cautions are placed in this guide to serve as warnings for certain actions.
 4. **For Windows:** Double-click the file to start the app.<br>
    **For Mac:** Open up a [terminal](#https://www.maketecheasier.com/launch-terminal-current-folder-mac/) in the current folder you have installed LinkedOUT. <br>
    Then, run the following command: <br>
-   ```java -jar LinkedOUT.jar```
+   ```java -jar LinkedOUT.jar``` 
+   
+    <br>
+   
+    LinkedOUT comes with sample data for you to experiment on. Upon starting the application, you should see the following screen: 
+   
+    <br>
```

Comment:
Not sure if this is intended but I think the spacing is slightly uneven here. Not sure is it because of <br> but maybe can consider removing it so that there wont be a gap before and after the line "Linkedout comes..."
![image](https://user-images.githubusercontent.com/77236381/162000540-47881742-747c-4d56-98d8-b13768601c27.png)

Reply:
Sure! Will remove the br